library/aes.c

```diff
@@ -1215,6 +1249,72 @@ static const unsigned char aes_test_cfb128_ct[3][64] =
 };
 #endif /* MBEDTLS_CIPHER_MODE_CFB */
 
+#if defined(MBEDTLS_CIPHER_MODE_OFB)
+/*
+ * AES-OFB test vectors from:
+ *
+ * http://csrc.nist.gov/publications/nistpubs/800-38a/sp800-38a.pdf
```

Comment:
Minor: for me today, this URL redirects to a webpage and the actual PDF is finally found at: https://nvlpubs.nist.gov/nistpubs/legacy/sp/nistspecialpublication800-38a.pdf

I'm not sure it's worth fixing, but if you want to change it, perhaps just refer to the document by its name, as NIST URLs tend to be far less permanent than I'd like them to be?

Reply:
The link in the source code goes to a landing page for the standard, not to a PDF, so hopefully is a bit more future proof.

Reply:
Ok. Since the URL ends in `.pdf` I assumed the intention was to point to the pdf directly, but apparently I misunderstood. Landing page is fine, but then why not point directly to https://csrc.nist.gov/publications/detail/sp/800-38a/final and save a redirect?

Anyway, this is minor and doesn't need to be addressed, the name of the publication should be enough for anyone to find it anyway.

Reply:
Ah. Hadn't realised the URL pointed to a .pdf. Have updated it to your proposed alternative (which seems as good as any).

Done.